### PR TITLE
try xUnit retry package

### PR DESCRIPTION
**Pull request by monsieurleberre**
Please don't shout, I am just doing this PR as a test to see if the CI gives better results

### CODE REVIEW

File: src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs [hunk 1]
```diff
@@ -194,7 +194,7 @@ public void Proof_of_work_node_calls_getstakinginfo_and_receives_error()
             Then(a_method_not_allowed_error_is_returned);
         }
 
-        [Fact]
+        [Retry]
```
Comment (dangershony) [inline src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs]: What is the retry default attempts?

**Comment by noescape00** [comment]
> I am just doing this PR as a test to see if the CI gives better results

But the CI doesn't work with integration tests because they are ignored

**Comment by bokobza** [comment]
I think it's ok to have a retry for now and remove it later. This will help us during the build to see the failures for what they really are.

File: src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs (hunk 1 above)
Comment (bokobza) [reply src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs]: it's 3 by default.

**Comment by noescape00** [comment]
Unit tests don't have severe issues that require retrying and integration tests are not ran by CI. 
I don't think we need retrying.

**Comment by bokobza** [comment]
There is no retry on unit tests and we want to run the integration tests from the CI.
Also, even locally, I think we'd benefit from having some tests retry by themselves.

**Comment by bokobza** [comment]
@dangershony also OK with this.

**Comment by noescape00** [comment]
>There is no retry on unit tests and we want to run the integration tests from the CI.
>Also, even locally, I think we'd benefit from having some tests retry by themselves.

Ok. Then it's ok as a temporary thing till we fix tests to be stable but in a long run I'd prefer not having this.

**Pull request merged**